tests/integration/helpers.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Helper functions for integration tests."""
+
+import typing
+
+import requests
+from juju.action import Action
+from juju.application import Application
+
+
+def create_moderators_room(
+    synapse_ip: str,
+    access_token: str,
+):
+    """Create "moderators" room in Synapse
+
+    Args:
+        synapse_ip: Synapse IP
+        access_token: Access token for Synapse to create the room
+
+    """
+    authorization_token = f"Bearer {access_token}"
```

Comment:
Perhaps `authorization_header`?

Reply:
I would rather leave that name as it it used in several places and I am not sure it is really wrong. Maybe it could be reasonable for another PR to look for other good names like [credentials](https://datatracker.ietf.org/doc/html/rfc6750#section-2.1) or "bearer authentication".